celery/app/base.py

```diff
@@ -475,6 +475,8 @@ def _task_from_fun(self, fun, name=None, base=None, bind=False, **options):
                 def run(*args, **kwargs):
                     try:
                         return task._orig_run(*args, **kwargs)
+                    except Ignore:
+                        raise
```

Comment:
@thedrow I think re-raising `Ignore` causes an maximum recursion condition. Perhaps simply `pass` here?
Also, if you can share the unit test, I might be able to take a look.

Reply:
I'll do so tomorrow.